internal/controller/workspace/workspace.go

```diff
@@ -159,40 +159,50 @@ func (c *connector) Connect(ctx context.Context, mg resource.Managed) (managed.E
 		return nil, errors.Wrap(err, errGetPC)
 	}
 
-	switch cr.Spec.ForProvider.Source {
-	case v1alpha1.ModuleSourceRemote:
-		// NOTE(ytsarev): Retrieve .git-credentials from Spec to /tmp outside of workspace directory
-		gitCredDir := filepath.Clean(filepath.Join("/tmp", dir))
-		if err := c.fs.MkdirAll(gitCredDir, 0700); err != nil {
-			return nil, errors.Wrap(err, errWriteGitCreds)
+	gitCredDir := filepath.Clean(filepath.Join("/tmp", dir))
```

Comment:
Nit: Are these always git credentials? Should this just be `credDir`?

Reply:
Also, it seems like we now write _all_ credentials files to this `credDir`. I believe previously we only wrote `.git-credentials` there, while other credentials files were written into the directory where we call `terraform init`. I think we still want to support that second use case.